aws/scenarios/dockerVM/run.go

```diff
@@ -3,9 +3,14 @@ package dockervm
 import (
 	ec2vm "github.com/DataDog/test-infra-definitions/aws/scenarios/vm/ec2VM"
 	"github.com/DataDog/test-infra-definitions/datadog/agent/docker"
+	"github.com/DataDog/test-infra-definitions/registry"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
 
+func init() {
+	registry.Scenarios.Register("aws/dockervm", Run)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
This is smart

Reply:
Interesting, as the only supposed caller is `main`, it can make sense to define them in `main`. I find it smart to have registration in the run functions, as you don't need to update `main` if you add a run function, as you'll be required to add `init`. Well it is not strictly required, but should be documented and by copy-pasting it is clear enough you need to register to use it.